addons/event/models/event_event.py

```diff
@@ -269,7 +269,8 @@ def _compute_event_registrations_open(self):
             event.event_registrations_open = (event.start_sale_date <= current_datetime.date() if event.start_sale_date else True) and \
                 (date_end_tz >= current_datetime if date_end_tz else True) and \
                 (not event.seats_limited or event.seats_available) and \
-                (not event.event_ticket_ids or any(ticket.sale_available for ticket in event.event_ticket_ids))
+                (not event.event_ticket_ids or any(ticket.sale_available for ticket in event.event_ticket_ids)) and \
+                (not event.stage_id.pipe_end)
```

Comment:
I don't understand the purpose of that change. Ended events are based on event end datetime, and tickets sale end datetime. Stage are there for organisational purpose in backend, not for frontend. Moreover events are moved automatically in "pipe end" stages when finished, not before.

Reply:
Hello @tde-banana-odoo I think the use case wasn't very clear, my bad for that. In the status bar of an event, it's possible to directly set the status to "Ended". The customer was trying to end the event by simply changing the stage instead of editing the ended date. However, the stage changing without editing the end date doesn't trigger the end of ticket selling, as you mentioned, the triggering only depends on those conditions.

I made this fix so that when the customer changes the event stage to  "Ended", the ticket cannot be sold anymore. Hope this explains a bit better :smile:

Reply:
Ok, I see the issue ... but I am not sure we have to do it, especially in stable. Indeed if I remember well, the "pipe end" trick was more a tool quickly done to automatically have events being dragged out of the "active" kanban view. Here the customer is doing the inverse move (moving the event to close it). It was not planned to block registrations, even if it could make sense.

As this is a functional change more than a bugfix, not sure what to do ... Maybe ask LNA what he thinks about that ? Then we will see if we target master or stable.

Cheers !

Reply:
I will ask him, thanks for checking! :smile:

Reply:
Hello @tde-banana-odoo I got a reply from the product owner today that the End stage is indeed just for organizational purpose and the clients shouldn't use it to end an event. I will close the PRs related to this task then. Thanks again for reviewing! :smile: